portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,73 @@
+
+- [Alle](#alle)
+- [UX](#ux)
+- [Utvikling](#utvikling)
+- [Test](#test)
+- [Produkt](#produkt)
+- [Support](#support)
+
+## Alle
+Det forventes at alle har en grunnlegende forståelse for hva universell utforming er og hva det betyr for oss som organisasjon. Det forventes også at det er fokus på temaet, ikke fordi det er lovpålagt, men fordi det er det gjør hos Fremtind. Å bygge kvalitetskravene inn i prosessen og produktene våre tar tid, og det viktig at alle er med å drar i riktig retning og forstår at dette tar litt tid å få til riktig.
+
+Det forventes:
+- Generell forståelse av hva universell utforming er
+- Forståelse av hva som forventes av de ulike rollene
+- Bidra til å sette fokus på universell utforming
+- Hjelpe hverandre til å øke forstålse og kunnskap
+
+## UX
+Designere og UXere er "førsteforsvaret" for universell utforming. Det er helt kritisk for å kunne levere en god opplevelse for brukerene vår at universell utforming er med fra første stund, det kan ikke være noe man legger på etter at man har designet løsningen. 
+
+Det forventes:
+- Forståelse av god informasjonsarktektur (kanskje vi burde lenke til noe i alle disse for å la folk fylle hull feks)
+- Forståelse av god fargebruk og kontrast
+- Forståelse av bevegelse og animasjon
+- Forståelse av hvordan designet blir oversatt til funksjonelle løsninger
+- Kunnskap om hvem våre brukere er og hvordan de bruker våre løsninger
+- Eksperter på brukertesting
+
+## Utvikling
+Utviklerne er ansvarlige for å oversette design og ideer til virkelighet, det innebærer å ha forståelese av hvordan designet blir i den virkelige verden på ulike skjermer, ulike dingser osv. Hvis man jobber med frontend i en eller annen form må man ta et stort ansvar for at våre løsninger skal være tilgjengelig for flest mulig. Man kan ikke gjemme seg bak "backend" eller "fullstack" merkelapp, jobber du med flater vi treffer kunder i, så er dette ditt ansvar.
+
+Det forventes:
+- Forståelse av semantisk HTML
+- Kunnskap om bruk av skjermleser og tastaturnavigasjon
+- Kunnskap om brukerpreferanser som redusert bevegelse
+- Kunnskap om responsiv CSS
+- Kunnskap om avanserte og/eller programatiske verifikasjonsverktøy
+- Automatisering av de deler av prosessen som kan automatiseres
+- Bry seg om brukeropplevelsen i sluttproduktet
+
+## Test
+Testerne er de som i størst grad representerer brukerne, derfor er det viktig at testerne evner å sette seg inn i en rekke forskjellige situasjoner og brukermønstre. Test er sikkerhetsnettet, men det teamet som er ansvarlig for kvaliteten på universell utforming, ikke test alene.
```

Comment:
Testerne er de som i størst grad representerer brukerne, <- punktum istedet for komma

"Test er sikkerhetsnettet, men det teamet som er ansvarlig for kvaliteten på universell utforming, ikke test alene." <- Er noe feil her.